webapp/src/containers/Rhs/index.tsx

```diff
@@ -104,8 +144,9 @@ const Rhs = (): JSX.Element => {
                             className='rhs-disconnect__disconnect-button'
                             onClick={() => {
                                 setDialogContent('Are you sure you want to disconnect your MS Teams Account?');
-                                setShowDisconnectDialog(true);
+                                setShowDialog(true);
                                 setShowDestructiveDialog(true);
+                                setPrimaryActionText('Disconnect');
```

Comment:
Constants for 'Unlink' and 'Disconnect' as they are being used at multiple places